src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetaDataTest.java

```diff
@@ -1195,6 +1197,334 @@ public void testJSONMetaData() throws SQLException {
         }
     }
 
+    /**
+     * Test to print all functions returned by getFunctions()
+     * 
+     * @throws SQLException
+     */
+    @Test
+    public void testPrintGetFunctionsResults() throws SQLException {
+        try (Connection conn = getConnection()) {
+            DatabaseMetaData metaData = conn.getMetaData();
+
+            System.out.println("\n=== getFunctions() Results ===");
+            System.out.println(
+                    "FUNCTION_CAT | FUNCTION_SCHEM | FUNCTION_NAME | NUM_INPUT_PARAMS | NUM_OUTPUT_PARAMS | NUM_RESULT_SETS | REMARKS | FUNCTION_TYPE");
+            System.out.println(
+                    "-------------|----------------|---------------|------------------|-------------------|------------------|---------|---------------");
+
+            int count = 0;
+            try (ResultSet rs = metaData.getFunctions(null, null, null)) {
+                while (rs.next()) {
+                    String functionCat = rs.getString("FUNCTION_CAT");
+                    String functionSchem = rs.getString("FUNCTION_SCHEM");
+                    String functionName = rs.getString("FUNCTION_NAME");
+                    int numInputParams = rs.getInt("NUM_INPUT_PARAMS");
+                    int numOutputParams = rs.getInt("NUM_OUTPUT_PARAMS");
+                    int numResultSets = rs.getInt("NUM_RESULT_SETS");
+                    String remarks = rs.getString("REMARKS");
+                    int functionType = rs.getInt("FUNCTION_TYPE");
+
+                    System.out.printf("%-12s | %-14s | %-13s | %-16d | %-17d | %-16d | %-7s | %d%n",
+                            functionCat != null ? functionCat : "NULL",
+                            functionSchem != null ? functionSchem : "NULL",
+                            functionName != null ? functionName : "NULL",
+                            numInputParams,
+                            numOutputParams,
+                            numResultSets,
+                            remarks != null ? remarks : "NULL",
+                            functionType);
+
+                    count++;
+                }
+            }
+            System.out.println("\nTotal functions found: " + count);
```

Comment:
It would be better to actually verify something...
(Add `import static org.junit.jupiter.api.Assertions.assertNotEquals;` to imports.)
```suggestion
            assertNotEqual(0, count);
```

Reply:
Removed this tests. It was just to verify the how result is returned.